WebRoot/messages/ZmMsg.properties

```diff
@@ -1463,6 +1463,7 @@ filterErrorCondition = The condition "{0}" is missing one or more values.
 filterErrorIllegalCharacter = Filter values may not contain double-quote (") or backslash (\\)
 filterErrorNameExists = There is already a filter by that name.
 filterErrorNoName = The new filter needs a name.
+filterErrorSieveScriptMaxSize = Filter size limit exceeded. Please examine the filter contents and reduce the number of settings.
```

Comment:
Will we translate it on the ticket or on a separate ticket?

Reply:
okay. Please create a ticket and write the ticket number here so that we will not miss it.